checks/raw/dependency_update_tool.go

```diff
@@ -127,6 +127,21 @@ var checkDependencyFileExists fileparser.DoWhileTrueOnFilename = func(name strin
 				},
 			},
 		})
+	// https://github.com/scala-steward-org/scala-steward/blob/main/docs/repo-specific-configuration.md
+	case ".scala-steward.conf",
+		"scala-steward.conf":
```

Comment:
you would also need the other 4 cases to handle the `.github` and `.config` folders:
>  The [.]scala-steward.conf configuration file can be located in the root of your repository, in .github directory or in .config directory (searched in this order).